sdk/python/pvsite_datamodel/read/latest_forecast_values.py

```diff
@@ -10,42 +10,46 @@
 
 
 def get_latest_forecast_values_by_site(
-    session: Session, site_uuids: List[uuid.UUID], start_utc: Optional[dt.datetime] = None
+    session: Session,
+    site_uuids: List[uuid.UUID],
+    start_utc: Optional[dt.datetime] = None,
+    model=LatestForecastValueSQL,
 ) -> Dict[uuid.UUID, List[LatestForecastValueSQL]]:
     """Get the latest forecast values by input sites.
 
     :param session: The sqlalchemy database session
     :param site_uuids: list of site_uuids for which to fetch latest forecast values
     :param start_utc: filters on forecast values target_time >= start_utc
+    :param model, the database table to use, could be 'LatestForecastValueSQL' or ForecastValueSQL
     :return: dict containing {site_uuid1: List[LatestForecastValueSQL], site_uuid2: ...}
     """
     # start main query
-    query: Query = session.query(LatestForecastValueSQL)
+    query: Query = session.query(model)
     query = query.join(DatetimeIntervalSQL)
 
     if start_utc is not None:
         query = query.filter(DatetimeIntervalSQL.start_utc >= start_utc)
 
         # also filter on creation time, to speed up things
         created_utc_filter = start_utc - dt.timedelta(days=1)
-        query = query.filter(LatestForecastValueSQL.created_utc >= created_utc_filter)
+        query = query.filter(model.created_utc >= created_utc_filter)
 
-    output_dict: Dict[uuid.UUID, List[LatestForecastValueSQL]] = {}
+    output_dict: Dict[uuid.UUID, List[model]] = {}
```

Comment:
Not a problem really but FYI:
You don't need to import `Dict` or `List`, `dict[uuid.UUID, list[model]]` is equivalent.